LDM-562.tex

```diff
@@ -0,0 +1,357 @@
+
+\textbf{Discussion:}
+It must be possible to use a trivial reference catalog with no more than two stars (which would constrain only the overall photometric zeropoint and the position, scale, and rotation of the image, requiring internal data to constrain all other terms).  Jointcal must be able to filter reference catalogs on magnitude and color and reject outliers (e.g. due to variability, poor external measurements, or different deblending).
+
+\subsection{Inputs from Colimated Beam Projector}
+
+\label{DMS-L2JC-REQ-0018}
+\textbf{ID:} DMS-L2JC-REQ-0018
+
+\textbf{Specification:}
+JointCal shall support using Collimated Beam Projector data to constrain the photometric fit.
+
+\textbf{Discussion:}
+CBP data consists of measurements of photometric throughput (with no atmospheric contributions) at a particular point on the focal plane at a particular wavelength. Like measurements of stars and galaxies, the overall flux of a particular CBP point is unknown but it can be considered to be constant over all exposures. Photometric fits that utilize CBP data only (i.e. no measurements on science images or reference catalogs) should also be possible for one-off diagnostic runs (i.e. this does not need to be part of normal production interface).
+
+\subsection{Maximum Problem Size}
+
+\label{DMS-L2JC-REQ-0019}
+\textbf{ID:} DMS-L2JC-REQ-0019
+
+\textbf{Specification:}
+JointCal shall be capable of fitting at least 6000 full LSST visits together.
+
+\textbf{Discussion:}
+This allows a sky area of approximately 2x2 focal planes at 10-year depth in all bands to be handled in a single consistent solution.
+
+\subsection{Support for Partial Focal Planes}
+
+\label{DMS-L2JC-REQ-0020}
+\textbf{ID:} DMS-L2JC-REQ-0020
+
+\textbf{Specification:}
+JointCal shall be capable of processing visits in which not some areas of the focal plane do not overlap any other visits in the set of visits being fit.
+
+\textbf{Discussion:}
+Visit- or detector-level parameters that only affect the model in areas with a small number of visits need not be fit.
+
+\subsection{Outlier Rejection}
+
+\label{DMS-L2JC-REQ-0021}
+\textbf{ID:} DMS-L2JC-REQ-0021
+
+\textbf{Specification:}
+JointCal shall be capable of rejecting bad astrometric and photometric measurements using the degree to which they disagree with the model.
+
+\textbf{Discussion:}
+This will probably require explicit guards against rejecting too many matches due to bad initial conditions.
+
+\subsection{Stellar Motion Fitting}
+
+\label{DMS-L2JC-REQ-0022}
+\textbf{ID:} DMS-L2JC-REQ-0022
+
+\textbf{Specification:}
+JointCal shall be capable of utilizing external reference catalog stars with nonzero proper motion and parallax.
+
+\textbf{Discussion:}
+We do not expect to need JointCal to be able to fit proper motion and parallax itself, though it should be capable of rejecting stars with significant motions that are not in the reference catalog.
```

Comment:
@RobertLuptonTheGood says:
> This seemed reasonable on first reading, but I'm not sure if we can produce usable secondary catalogues that assume that stars don't move over 10 years.

I don't think the jointcal design precludes fitting proper motion and parallax (and I don't it to expect in a way that would), but I think we could also address this by extending our "primary" references by using previous LSST data release catalogs.